pytravharv/WebAccess.py

```diff
@@ -43,7 +44,7 @@ def handle_data(self, data):
             self.scripts.append({self.type: data})
 
 
-def WebAccess(url):
+def fetch(url):
```

Comment:
as mentioned before -- slightly prefer `find_description_graph(subject)`

also type the arguments and return in the signature

```
def find_description_graph(subject: str) -> Graph:
```